homework-assignments-1/task-1.js

```diff
@@ -0,0 +1,35 @@
+class Shape {
```

Comment:
I understand you did homework in js but this should be abstract class and abstract method. Please update